src/content/docs/dev/general-message-passing/xrpl/xrpl-gmp.mdx

```diff
@@ -0,0 +1,196 @@
+# XRPL General Message Passing (GMP)
+
+The XRPL Blockchain is unique compared to other blockchains integrated with Axelar. Unlike other blockchains, XRPL does not support smart contracts. As such there is no Axelar Gateway and Axelar Gas Service deployed on the chain the way there is with other EVM and non-EVM blockchain integrations. Since XRPL has no contracts there is also no ability to receive GMP messages from other blockchains, you can only send messages to other blockchains.
+
+## XRPL Multisig
+
+The integration leverages [XRPL Multisig Signing](https://xrpl.org/docs/concepts/accounts/multi-signing) to facilitate message passing. The [multisig](https://github.com/commonprefix/axelar-amplifier/blob/xrpl/contracts/xrpl-multisig-prover/src/xrpl_multisig.rs) is controlled by the [Axelar Verifier set](https://axelarscan.io/verifiers).  
+
+The testnet address for the XRPL multisig is: [rNrjh1KGZk2jBR3wPfAQnoidtFFYQKbQn2](https://testnet.xrpl.org/accounts/rNrjh1KGZk2jBR3wPfAQnoidtFFYQKbQn2).
+
+The mainnet address for the XRPL multisig is: [rfmS3zqrQrka8wVyhXifEeyTwe8AMz2Yhw](https://livenet.xrpl.org/accounts/rfmS3zqrQrka8wVyhXifEeyTwe8AMz2Yhw).
+
+## Send GMP Message
+
+### Overview
+
+The flow for sending a GMP message from XRPL involves sending a [payment transaction](https://xrpl.org/docs/references/protocol/transactions/types/payment) from your XRPL account to the XRPL multisig with a specified [memo](https://xrpl.org/docs/references/protocol/transactions/common-fields#memos-field) field. Once the multisig receives that transaction, Axelar's relayer will pickup any transaction that it has received with the `call_contract` message type specified at which the point the [Amplifier GMP flow](/dev/amplifier/introduction/#general-message-passing-with-amplifier) will commence. 
+
+### Payment Transaction
+
+Payment transaction types in XRPL represent a transfer of value. They require the following fields:
+
+1. `Transaction Type`: The [type](https://xrpl.org/docs/references/protocol/transactions/types) of XRPL transaction being sent.
+1. `Account`: The sender's account address.
+1. `Amount`: The amount of XRP being sent to cover the transaction fees.
+1. `Destination`: The destination of the transaction.
+1. `Memos`: A special field in the payment transaction that is used to specify the message type and the destination chain. 
+
+### Memo
+
+When sending a GMP message the `memo` field contains the relevant metadata for the transaction.
+
+The memo must contain the following fields:
+
+1. `type`: The type of transaction being sent (in this case, it should be of type `call_contract`).
+1. `destination_address`: The address of the destination contract on the destination chain.
+1. `destination_chain`: The name of the destination chain.
+1. `payload`: The payload of the transaction being sent.    
```

Comment:
this is optional right?

Reply:
The payload? I believe it is but in the context of making a GMP call you do need to have a payload otherwise youre not really making a GMP call